src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -399,24 +397,15 @@ void ProcessScopeForIndividualColumns(LogRecordScope scope, object state)
                 }
             }
 
-            logRecord.ForEachScope(ProcessScopeForEnvProperties, (object)null);
-            void ProcessScopeForEnvProperties(LogRecordScope scope, object state)
-            {
-                foreach (KeyValuePair<string, object> scopeItem in scope)
-                {
-                    if (string.IsNullOrEmpty(scopeItem.Key) || scopeItem.Key == "{OriginalFormat}")
-                    {
-                        continue;
-                    }
+            // Prepare state for scopes
+            dataForScopes.Cursor = cursor;
+            dataForScopes.EnvPropertiesCount = envPropertiesCount;
```

Comment:
Do we need to reset `HasEnvProperties` here? Looks like the scope delegate doesn't always set it.

Reply:
Maybe also `FieldsCount`

Reply:
Maybe up around Line 367.

Reply:
Updated.